spec/unit/provider/user/linux_spec.rb

```diff
@@ -34,15 +34,18 @@
     "uid" => "-u",
     "shell" => "-s",
     "password" => "-p",
+    "expire_date" => "-e",
+    "inactive" => "-f",
   }
 
   include_examples "a useradd-based user provider", supported_useradd_options
 
+  before(:each) do
+    @new_resource = Chef::Resource::User::LinuxUser.new("adam", @run_context)
+    @current_resource = Chef::Resource::User::LinuxUser.new("adam", @run_context)
+  end
```

Comment:
the `before` needs to be inside of a `describe` block, so just copy this into each of the 3 describes.

Reply:
fixed